packages/react/src/components/form/Input/Input.tsx

```diff
@@ -7,7 +7,7 @@ type InputAttr = InputHTMLAttributes<HTMLInputElement>;
 export type InputProps = {
   /** Supported `input` types */
   type?: InputAttr['type'];
-  /** Defines the width of <Input> in count of characters.
+  /** Defines the width of input in count of characters.
    */
   size?: number;
```

Comment:
Can't this be omitted, since it comes with the input props?

Reply:
Yeah, just have experienced developers are not very good at using the `size` attribute, and instead goes about using `style={{ width:  }}` or similar, so just wanted to highlight the attribute in our docs :)

Reply:
Fair enough, does not hurt to keep it, as long as we are consistent with it.